driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedHelper.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.unified;
+
+import org.bson.BsonDocument;
+
+abstract class UnifiedHelper {
+
+    static Long getAndRemoveTimeoutMS(final BsonDocument arguments) {
```

Comment:
If this is only used in `UnifiedGridFSHelper` lets just add it there.

Reply:
This method is used in both `UnifiedGridFSHelper` and `UnifiedCrudHelper`.